modules/protobuf/26.0.bcr.1/MODULE.bazel

```diff
@@ -0,0 +1,24 @@
+module(
+    name = "protobuf",
+    version = "26.0.bcr.1",
+    compatibility_level = 1,
+    repo_name = "com_google_protobuf",
```

Comment:
adding this is the change. required by this https://github.com/protocolbuffers/protobuf/blob/8d025c08ff9c4e63f37f62b64f3046183ef7ab20/upb_generator/bootstrap_compiler.bzl#L30